### PR TITLE
some fixes in spanish translation

### PR DESCRIPTION
**Pull request by PCTroller**
Added "amount_hours" and "amount minutes" that wasn't in actual code.

Fixed play time showing {cantidad} as string instead of showing the amount you play the game

### CODE REVIEW

File: src/locales/es/translation.json [hunk 1]
```diff
@@ -62,14 +62,16 @@
     "publisher": "Publicado por: {{publisher}}",
     "copy_link_to_clipboard": "Copiar enlace",
     "copied_link_to_clipboard": "Enlace copiado",
-    "hours": "horas",
-    "minutes": "minutos",
+    "hours": "hours",
+    "minutes": "minutes",
```
Comment (zamitto) [inline src/locales/es/translation.json:66]: Are these changes intended?

Comment (PCTroller) [reply src/locales/es/translation.json]: it wasn't, some missclick i think, i rewrite them in spanish

**Review by zamitto — APPROVED** [review]
